### PR TITLE
Sidebar.

### PR DESCRIPTION
**Pull request by Nowembabazi**
# I achieved the following.
Added styles .css
Added index.html
Added linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    Great Actor and a mentor for many young actors.
+                    Boseman was also a powerful screen presence playing the baseball player Jackie Robinson
+                    And the civil rights lawyer Thurgood Marshall. 
+                </h4>
+
+            </div>
+        </div>
+        <hr>
+ 
+        <div id="tribute-info">
```
Comment (Nowembabazi) [inline index.html:39]: Please try soo much to use minimal IDs in your work

File: index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                <br><br> ☛ Boseman died at his Los Angeles home as a result of complications related to colon cancer on August 28, 2020, with his wife and family by his side.[159][160][161] He was 43 years old.
+                <br>
+            </p>
+ 
+ 
+        </div>
+  
+        <br>
+            For more information,
+        check
```
Comment (Nowembabazi) [inline index.html:64]: Your information bar is not under any type of tag, please use a p tag or an h6 tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  
+        <br>
+            For more information,
+        check 
+        <a id="tribute-link" href="#"target= "_blank">
+            <b> ChadWick Boseman</b> on Wikipedia. 
+            <h5>Developed by @<a href="#">
+                    Nickson.</a></a>
+            </h5>
+      <main id="main">
```
Comment (Nowembabazi) [inline index.html:70]: Please remove the bottom main tag and the Id they seem to be covering nothing.

**Review by Nowembabazi — COMMENTED** [review]
# Required changes.
Please work on the required changes and submit your work for further review.

Thank you so much

**Review by sumairq — APPROVED** [review]
### Hi @Nowembabazi   , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Great Communication
✔️ Friendly tone
✔️Made good inline comments🥇
✔️Awesome review summary

## Optional suggestions

- [Optional] It's always nice to make generous use of emojis 😃 in your review comments to keep the tone less authoritative and more friendly. Cheers 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**